apps/src/music/player/sequencer/Simple2Sequencer.ts

```diff
@@ -48,6 +48,10 @@ export default class Simple2Sequencer extends Sequencer {
   private startMeasure: number;
   private inTrigger: boolean;
 
+  // A set of strings (e.g. "electro/beat-4", which is an event ID and when to play) that are used
```

Comment:
Nit: I initially read the entire "electro/beat-4" as the name of the sound and didn't realize the 4 was related to when it plays. Could this comment say "which is a hyphen-separated event ID and measure"?

Reply:
Updated.